profiler/static/src/js/tour.js

```diff
@@ -0,0 +1,74 @@
+odoo.define("profiler.tour", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var tour = require("web_tour.tour");
+
+    var _t = core._t;
+
+    tour.register(
+        "profiler_tour",
+        {
+            url: "/web",
+        },
+        [
+            tour.STEPS.SHOW_APPS_MENU_ITEM,
```

Comment:
Getting `Error while loading profiler.tour: TypeError: Cannot read properties of undefined (reading 'SHOW_APPS_MENU_ITEM')`

May have to be rewritten as [example](https://github.com/OCA/OCB/blob/14.0/addons/sale/static/src/js/tours/sale.js#L13)